xapian-applications/omega/postprocess

```diff
@@ -201,12 +202,6 @@ def test_functions():
                     assert row == ['098f6bcd4621d373cade4e832627b4f6',
                                   'test', '35,47', '0', '35:1,47:0'], "Incorrect entry in final.log"
 
-        test_query = tempfile.NamedTemporaryFile(delete=True)
-
-        generate_query_file(test_final.name, test_query.name)
-
-        test_query.flush()
-
```

Comment:
Made this change to resolve `test_query` variable scope error (`UnboundLocalError: local variable test_query referenced before assignment`) at `test_query.flush()` being reported when one of the assert statements fails for final log entries above.